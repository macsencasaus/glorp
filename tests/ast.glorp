#!/bin/sh
exec ./glorp -a "$0"

1 + 2

##############
# NOTE: the following assertions are auto-generated by test.py
#
# PROGRAM(1)
#     INFIX EXPRESSION
#         OP: '+'
#         LEFT:
#             INT LITERAL 1
#         RIGHT:
#             INT LITERAL 2
