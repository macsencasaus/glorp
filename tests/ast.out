PROGRAM(1)
    INFIX EXPRESSION
        OP: '+'
        LEFT:
            INT LITERAL 1
        RIGHT:
            INT LITERAL 2
